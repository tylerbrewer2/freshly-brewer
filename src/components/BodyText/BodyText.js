import React from 'react';
import styled from 'styled-components';

const BodyTextCore = ({ className, children }) => {
  return (
    <p className={className}>
      {children}
    </p>
  )
}

const BodyText = styled(BodyTextCore)`
  font-family: 'AvenirNextRegular', 'Helvertica', 'sans-serif';
  font-size: 14px;
  line-height: 25px;
`

export default BodyText;
