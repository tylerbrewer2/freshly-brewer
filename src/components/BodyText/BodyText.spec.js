import React from 'react';
import renderer from 'react-test-renderer';

import BodyText from './BodyText'

it('renders correctly', () => {
  const tree = renderer.create(
    <BodyText>
      Hello
    </BodyText>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});