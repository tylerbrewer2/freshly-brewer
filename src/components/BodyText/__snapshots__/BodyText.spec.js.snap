exports[`test renders correctly 1`] = `
<p
  className="sc-bdVaJa eGZoUg">
  Hello
</p>
`;
